django_docker_engine/container_managers/docker_engine.py

```diff
@@ -26,6 +27,9 @@ def __init__(self, client=docker.from_env(), pem='django_docker_cloudformation.p
     def run(self, image_name, cmd, **kwargs):
         return self._containers_client.run(image_name, cmd, **kwargs)
 
+    def pull_image(self, image_name):
```

Comment:
The internal method is just `pull`, and there's nothing we need to disambiguate. Could the two new methods also just be `pull`?